packages/grafana-ui/src/components/DataLinks/DataLinkInput.tsx

```diff
@@ -124,7 +124,6 @@ export const DataLinkInput: React.FC<DataLinkInputProps> = memo(
           'gf-form-input',
           css`
             position: relative;
-            height: auto;
```

Comment:
Just check if this alters anything in panel data links

Reply:
Yes, I have checked the Panel Links and all looks good. 🙂